test/blackbox/common/BlackboxTestsDiscovery.cpp

```diff
@@ -223,6 +223,92 @@ TEST(Discovery, StaticDiscovery)
     reader.block_for_all();
 }
 
+TEST(Discovery, StaticDiscoveryFromString)
```

Comment:
I'd like a description of the general purpose of the tests followed by a step by step guide on the actions taken